docs/builds.md

```diff
@@ -1,4 +1,26 @@
-
+<!--
+* Copyright (c) 2021, 2022 IBM Corp. and others
```

Comment:
This should say '2017, 2022' if we're following the guidelines.

Reply:
Yep, done, thanks